### ISSUE TITLE
Chrashes during jailbreak on macOS Big Sur

### ISSUE THREAD

**Issue by zx331645** [NSSpiral/Blackb0x#44]
I'm trying to jailbreak my AppleTV2,1 but blackb0x crashes right at the end of the process. I've downgraded to 5.3 already.

**Comment by perampere** [comment]
I got the same issue here.
ATV downgraded to 5.3
Power cable connected
Blackb0x crashes during reboot of ATV.
After that, Backb0x crashes everytime at startup if ATV is connected.

**Comment by Tgm77** [comment]
Same problem as well.

**Comment by mcjimsr** [comment]
> 
> 
> I got the same issue here.
> ATV downgraded to 5.3
> Power cable connected
> Blackb0x crashes during reboot of ATV.
> After that, Backb0x crashes everytime at startup if ATV is connected.

I have found if you manually reset the ATV, using Locations in Big Sur (iTunes replacement) it starts working again and does not crash. It only crashes when an ATV is connected mid exploit I'd imagine

**Comment by zx331645** [comment]
Yeah, but when i jb again it crashes again and i have to reset it again.

**Comment by renatoDutraSilva** [comment]
having the exact same problem. app crashes during jailbreak.


Process:               Blackb0x [10341]
Path:                  /private/var/folders/*/Blackb0x.app/Contents/MacOS/Blackb0x
Identifier:            spiral.Blackb0x
Version:               1.0 (1)
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           Blackb0x [10341]
User ID:               501

Date/Time:             2021-08-24 13:13:25.233 +0000
OS Version:            macOS 11.5.2 (20G95)
Report Version:        12
Bridge OS Version:     5.5 (18P4759a)
Anonymous UUID:        E6F7A94C-289A-E7F8-25BF-78BA45721D3B

Sleep/Wake UUID:       E52102AB-387B-44A5-9DC3-E3652748F3DB

Time Awake Since Boot: 27000 seconds
Time Since Wake:       610 seconds

System Integrity Protection: enabled

Notes:                 Translocated Process

Crashed Thread:        4  Dispatch queue: com.apple.root.default-qos

Exception Type:        EXC_BAD_ACCESS (SIGSEGV)
Exception Codes:       KERN_INVALID_ADDRESS at 0x0000000000000028
Exception Note:        EXC_CORPSE_NOTIFY

Termination Signal:    Segmentation fault: 11
Termination Reason:    Namespace SIGNAL, Code 0xb
Terminating Process:   exc handler [10341]

**Comment by zx331645** [comment]
@NSSpiral Please help, i've been trying for months.
